src/main/java/software/amazon/smithy/lsp/ext/FileCachingCollector.java

```diff
@@ -274,7 +274,9 @@ private Boolean shapeWasDefinedInline(OperationShape operation, Shape shape, Mod
         }
         int boundary = Math.max(priorShapeLine, operation.getSourceLocation().getLine());
         while (shapeStartLine >= boundary) {
-            String line = modelFile.lines.get(shapeStartLine);
+            String line = modelFile.lines.get(shapeStartLine - 1);
+
+            // note: this doesn't take code inside comments into account
```

Comment:
This might be an issue but I haven't gotten around to provoking it yet.

Reply:
Are you planning to test further to see if this is an issue?